ecosystem/indexer-grpc/indexer-grpc-utils/src/cache_operator.rs

```diff
@@ -186,34 +213,101 @@ impl<T: redis::aio::ConnectionLike + Send + Clone> CacheOperator<T> {
             // TODO: rewrite this logic to surface this max fetch size better
             Ok(CacheCoverageStatus::CacheHit(std::cmp::min(
                 latest_version - requested_version,
-                BLOB_STORAGE_SIZE as u64,
+                FILE_ENTRY_TRANSACTION_COUNT,
             )))
         }
     }
 
+    /// Fail if not all transactions requested are returned
+    pub async fn batch_get_encoded_proto_data_x(
```

Comment:
is this used anywhere?

Reply:
yea; rename it to something meaningful. we'll address it with refactoring.